### ISSUE TITLE
Skeleton should track method-method interactions

### ISSUE THREAD

**Issue by skapral** [yegor256/jpeek#106]
...like methods calling each other.

Steps to reproduce:

1. Create a project with a single class in there:
```
public class Proba {
    private final int a;
    private final int b;

    public Proba(final int a, final int b) {
        this.a = a;
        this.b = b;
    }

    public final int sum() {
        return a + b;
    }

    public final int sub() {
        return a - b;
    }

    public final int mul() {
        return sum() * sub();
    }
}
```
Note: method `mul` calls `sum` and `sub`, which creates additional relations between them.

2. Analyse it with `jpeek` tool (see README.md)

3. Check generated `skeleton.xml`

```
<class id="Proba">
            <attributes>
               <attribute public="false" static="false" type="I">a</attribute>
               <attribute public="false" static="false" type="I">b</attribute>
            </attributes>
            <methods>
               ...
               <method abstract="false"
                       ctor="false"
                       desc="()I"
                       name="mul"
                       public="true"
                       static="false">
                  <return>I</return>
                  <args/>
               </method>
            </methods>
         </class>
```

Usually, interactions betweeen elements of a class are in `<ops/>` section. For `mul` method there is nothing.

This information is crucial for gathering a set of metrics based on disjoints or decompositions, for instance LCC and (I)CBMC.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#106](https://github.com/yegor256/jpeek/issues/106) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://datum.zerocracy.com/pages/policy.html#29): +15 points just awarded to @skapral[/z](https://www.0crat.com/u/skapral), total is [+45](https://www.0crat.com/u/skapral)

**Comment by 0crat** [comment]
The job [#106](https://github.com/yegor256/jpeek/issues/106) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale). The budget is 30 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§8](http://datum.zerocracy.com/pages/policy.html#8) and [§9](http://datum.zerocracy.com/pages/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by llorllale** [comment]
@skapral would this suffice?

```xml
<op type="call">java.lang.String.substring</op>
```

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [§8](http://datum.zerocracy.com/pages/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by llorllale** [comment]
ping @skapral

**Comment by skapral** [comment]
@llorllale no objections

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) this job was assigned to you 8 days ago. It will be taken away from you soon, unless you close it, see [§8](http://datum.zerocracy.com/pages/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by llorllale** [comment]
@0crat waiting

**Comment by 0crat** [comment]
> @0crat waiting ([here](https://github.com/yegor256/jpeek/issues/106#issuecomment-364759764))

@llorllale The impediment for [#106](https://github.com/yegor256/jpeek/issues/106) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by llorllale** [comment]
blocked by #170

**Comment by llorllale** [comment]
blocked by #172

**Comment by llorllale** [comment]
@yegor256 fixed in #153. Can we close this?

**Comment by llorllale** [comment]
Sorry, @skrapal this was fixed in #153. Can we close this?

**Comment by llorllale** [comment]
ping @skapral

**Comment by skapral** [comment]
@llorllale ok, let's close.

**Comment by 0crat** [comment]
Order was successfully finished: +30 points just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale), total is [+555](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#106](https://github.com/yegor256/jpeek/issues/106) is now out of scope

**Comment by 0pdd** [comment]
@skapral the only puzzle [#176](https://github.com/yegor256/jpeek/issues/176) is solved here.
